con-base/src/main/java/ba/codecentric/base/service/ImageService.java

```diff
@@ -0,0 +1,8 @@
+package ba.codecentric.base.service;
+
+import org.springframework.web.multipart.MultipartFile;
+
+public interface ImageService {
+
+    String saveImage(MultipartFile image);
```

Comment:
`MultipartFile` shouldn't be used in the service layer since it belongs to Spring web (controller layer). Changing framework or HTTP toolkit in controller layer would probably break your service layer. You probably want to pass `InputStream` (which can be obtained from `MultipartFile`) and file name. String return type doesn't say anything. Maybe boolean? Add Java docs above that method.